api/v1alpha1/ndb_api_helpers.go

```diff
@@ -263,13 +340,18 @@ func GetActionArgumentsByDatabaseType(databaseType string) (DatabaseActionArgs,
 	case DATABASE_TYPE_MONGODB:
 		dbTypeActionArgs = &MongodbActionArgs{}
 	default:
-		return nil, errors.New("Invalid Database Type: supported values: mysql, postgres, mongodb")
+		return nil, errors.New("invalid database type: supported values: mysql, postgres, mongodb")
 	}
 	return dbTypeActionArgs, nil
 }
 
 func (m *MysqlActionArgs) GetActionArguments(dbSpec DatabaseSpec) []ActionArgument {
-	return []ActionArgument{}
+	return []ActionArgument{
+		{
```

Comment:
will discuss these changes and remove them. I guess they were pulled in because of the rebase gone wrong.